.github/workflows/pull-request.yml

```diff
@@ -28,7 +31,7 @@ jobs:
       matrix:
         include:
           - os: linux
-            run: ubuntu-22.04
+            run: ubuntu-latest
           - os: darwin
```

Comment:
Can you comment out `darwin` here to avoid adding unexpected costs? People can then uncomment it an manually run selecting their local fork/branch  if they want, but it then will be opt-in.

Reply:
If it's a public repo, it's free, according the the [Billing page](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions).

If you're still worried, I can easily comment it out, both in `pull-request`, and over in the `manual-go-test` workflow as well.

Reply:
I've pushed up a commit disabling the MacOS tests.